uploader/pantheon.py

```diff
@@ -161,6 +157,146 @@ def remove_trailing_slash(path):
         path = path[:-1]
     return path
 
+def recursive_glob(directory, pattern='*'):
```

Comment:
Hi Lisa, my suggestion with methods like these is to add some documentation strings (or whatever they are called in python) so that it's easier to follow the review.

Reply:
Carlos,

I updated the new methods with docstrings.